app/scripts/modules/core/src/managed/ManagedResourceStatusIndicator.tsx

```diff
@@ -177,30 +185,125 @@ const logClick = (label: string, resourceId: string, status: ManagedResourceStat
     label: `${resourceId},${status}`,
   });
 
-const LearnMoreLink = ({ id, status }: { id: string; status: ManagedResourceStatus }) => (
-  <p className="sp-margin-m-top sp-margin-xs-bottom">
-    <a
-      target="_blank"
-      onClick={() => logClick('Status docs link', id, status)}
-      href={`https://www.spinnaker.io/reference/managed-delivery/resource-status/#${status.toLowerCase()}`}
-    >
-      Learn more about this
-    </a>
-  </p>
+const toggleResourcePause = (
```

Comment:
thoughts on making the whole non-popover flow it's own module/component? for now, could just expose some sort of function like `showResourcePauseModal({ resourceSummary, application})` and then maybe call `hidePopover` in the `onClick` for the toggle button? It feels like it's own thing, and this file is starting to get unruly.

Reply:
yeah, will do